app/controllers/concerns/api/follows_controller.rb

```diff
@@ -16,5 +23,19 @@ def tags
         .includes(:followable)
         .order(points: :desc)
     end
+
+    private
+
+    def user_ids
+      return [] if params[:users].blank?
+
+      @user_ids ||= params[:users].pluck("id")
+    end
+
+    def org_ids
```

Comment:
Potential follow-up work here could be to add organizations params support to the API documentation.